vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,19 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting
+	Output     string // Output from last check
```

Comment:
I really don't understand what the purpose of limiting this is. You're overcomplicating the validation and support when a simple `[]byte` will take care of most of this. Sure output can be truncated, but it is senseless for us to care about the encoding.